### ISSUE TITLE
PATH user environment variable ignored on windows.

### ISSUE THREAD

**Issue by noc7c9** [xonsh/xonsh#2341]
From what I understand windows is supposed to append the user PATH variable to the system PATH and expose that to processes. But xonsh seems to be using the system PATH by itself, completely ignoring the user PATH.

```
user@athir-laptop P:\gdrive
$ $XONSH_VERSION
'0.5.9'

user@athir-laptop P:\gdrive
$ len(os.environ['PATH'].split(';'))
44

user@athir-laptop P:\gdrive
$ len($PATH)
38

user@athir-laptop P:\gdrive
$ set(os.environ['PATH'].split(';')) - set($PATH)
{'C:\\Users\\user\\.cargo\\bin',
 'C:\\Users\\user\\.lein\\bin',
 'C:\\Users\\user\\AppData\\Local\\.meteor\\',
 'C:\\Users\\user\\AppData\\Roaming\\cabal\\bin',
 'C:\\Users\\user\\AppData\\Roaming\\local\\bin',
 'C:\\Users\\user\\AppData\\Roaming\\npm'}
```
The output of that last command is identical to my user PATH.

**Comment by melund** [comment]
Thanks for reporting @noc7c9 ... It is a bug, see this prior  bug report #2320 .  Should be fixed by #2332 on master. Would you mind testing if it fixes it for you?

**Comment by noc7c9** [comment]
It indeed seems to be fixed, thanks.
```
user@athir-laptop P:\gdrive
$ set(os.environ['PATH'].split(';')) - set($PATH)
set()
```
